test/functional/zerocoin_wrapped_serials.py

```diff
@@ -91,7 +91,7 @@ def run_test(self):
                 tx = self.node.spendrawzerocoin(serial, randomness, DENOM_TO_USE, privkey)
             except JSONRPCException as e:
                 exc_msg = str(e)
-                if exc_msg == "The new spend coin transaction did not verify (-4)":
+                if exc_msg == "CoinSpend: failed check (-4)":
```

Comment:
Intentional to include this in the PR?

Reply:
yes, this is the resulting string that the wallet outputs being checked against